srcpkgs/hosts-gen/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'hosts-gen'
+pkgname=hosts-gen
+version=0.9
+revision=1
+build_style=gnu-makefile
+hostmakedepends="git"
+short_desc="Framework to generate /etc/hosts from various sources"
+maintainer="mustaqim <mustaqim+void@pm.me>"
+license="GPL-3.0-or-later"
+homepage="http://r-36.net/scm/hosts-gen"
+changelog="http://r-36.net/scm/hosts-gen/log.html"
+patch_args="-Np1"
+
+do_fetch() {
+	git clone git://r-36.net/hosts-gen ${pkgname}-${version}
```

Comment:
No, we need a release not a git package.

Reply:
The upstream repo allows for creating a release tarball with `make dist`, it doesn't provide one.